.github/workflows/ci.yml

```diff
@@ -0,0 +1,17 @@
+name: test
+on:
+  push:
+    branches:
+      - "main"
+  pull_request_target:
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: “3.10”
```

Comment:
I don't think these quotes will be accepted.

Reply:
The quotes are VITAL in YAML.  This is the Yaml Norway problem.

https://dev.to/hugovk/the-python-3-1-problem-85g